.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "oni-lda-c"]
-	path = oni-lda-c
-	url = https://github.com/OpenNetworkInsight/oni-lda-c
```

Comment:
hell, in my current repo I can't even add this file to git, it's being ignored...
something is weird